pyethapp/config.py

```diff
@@ -103,13 +115,14 @@ def _fix_accounts(path):
         write_config(config)
 
 
-def load_config(path=default_config_path):
+def load_config(path=default_config_path, fix_accounts=True):
```

Comment:
What should be the content of existing data dir if any? In the test I have to create that content and need to know what to create.